src/components/AccountAddress.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+import React, { Component, PropTypes } from 'react'
+import { Text } from 'react-native'
+import AppStyles from '../styles'
+
+export default class AccountAddress extends Component {
+  static propTypes = {
+    address: PropTypes.string.isRequired
+  }
+
+  state = {
+    address: this.props.address
+  }
+
+  componentWillReceiveProps (newProps) {
```

Comment:
Why do you copy `props` to `state`? Doesn't seem to process those in any way, so I think it's superfluous.